LiteFarmOrg/LiteFarm
# Lf 2818 : refactor custom field work tasks to mirror pattern for harvest task

To Test: 

1. run the migration script 
2. go to the field task page
3. select other as the task type and add the task
4. go again to the field task page and check the custom task getting added to the drop-down.
